core/src/main/java/org/mule/transformer/simple/ObjectToString.java

```diff
@@ -85,6 +85,7 @@ protected String createStringFromInputStream(InputStream input, String outputEnc
             try
             {
                 input.close();
+                input.reset();
```

Comment:
you can't do this.

1) It makes no sense to reset a closed stream. With a simple ByteArrayInputStream it might work, but with an InputStream coming from ftp, http, sockets, etc. there're no guarantees. 
2) You're violating the reset() contract. This method only works on SOME input streams and provided the mark() method was previously called. You can't assume either here
3) Why did you do this in the first place? Looks like there's some issue that's been hidden rather than fixed